src/themes/teams/components/Segment/segmentVariables.ts

```diff
@@ -0,0 +1,18 @@
+import { pxToRem } from '../../../../lib'
+import { ComponentVariablesPrepared } from '@stardust-ui/react'
+
+export interface SegmentVariables {
+  padding: string
+  color: string
```

Comment:
`color` and `fontSize` are not used anywhere in segmentStyles

Reply:
thanks, fixed